.github/stale.yml

```
@@ -0,0 +1,56 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# General configuration
+
+# Pull request specific configuration
+pulls:
+  staleLabel: awaiting changes
+  # Number of days of inactivity before an Issue or Pull Request becomes stale
+  daysUntilStale: 45
+  # Number of days of inactivity before a stale Issue or Pull Request is closed.
+  # Set to false to disable. If disabled, issues still need to be closed manually, but will remain marked as stale.
+  daysUntilClose: 30
+  # Comment to post when marking as stale. Set to `false` to disable
+  markComment: >
+    This pull request has been automatically marked as stale because it has not had
+    activity in the last 45 days. It will be closed in 30 days if no further activity occurs.
+    Please feel free to give a status update now, or re-open when it's ready.
+
+    Thank you for your contributions!
+  # Comment to post when closing a stale Issue or Pull Request.
+  closeComment: >
+     This pull request has been automatically closed because it has not had activity in the last 30 days.
+     Please feel free to give a status update now, ping for review, or re-open when it's ready.
+
+     Thank you for your contributions!
+  # Limit the number of actions per hour, from 1-30. Default is 30
+  limitPerRun: 30
+  exemptLabels:
+    - awaiting review
+    - breaking_change
+    - in progress
+    - on hold
+
+# Issue specific configuration
+issues:
+  staleLabel: solved
+  limitPerRun: 10
+  daysUntilStale: 90
+  daysUntilClose: 30
+  markComment: >
+    This issue has been automatically marked as resolved because it has not had activity in the
+    last 90 days. It will be closed in the next 30 days unless it is tagged properly or other activity
+    occurs.
+
+    Thank you for your contributions!
+  closeComment: >
+    This issue has been automatically closed because it has not had activity in the last 30 days.
+    If this issue is still valid, please ping a maintainer.
```

```suggestion
    If this issue is still valid, please ping one of the Collaborators.
```

Or do we want this to include keyboard maintainers, in the case that the issue is about a specific board?

---

I wasn't sure about the best wording here.  Which is why I went with "maintainer"